src/analysis/thermo/vonBertalanffy/initVonBertalanffy.m

```diff
@@ -194,4 +62,3 @@
     setenv('LD_LIBRARY_PATH',['/usr/lib/x86_64-linux-gnu:' getenv('LD_LIBRARY_PATH')]);
```

Comment:
This instruction is only for Linux operating system. Thus this function should include a check on the OS and fail gracefully if it is not a Linux OS.

Reply:
Ok, I'll change it in the other branch where I work on the tutorial file